config/mfDefsGEM.mk

```diff
@@ -248,8 +257,13 @@ GCCVERSION = $(shell $(CC) -dumpversion)
 #NM     = llvm-nm
 #RANLIB = llvm-ranlib
 # CLANGVERSION = $(shell $(CC) --version | awk '/clang /{print $0;exit 0;}')
-CLANGVERSION = $(shell $(CC) -dumpversion)
+CLANGVERSION = $(shell $(CC) -dumpfullversion -dumpversion)
 
-CCVERSION = $(shell $(CC) -dumpversion)
+CCVERSION = $(shell $(CC) -dumpfullversion -dumpversion)
```

Comment:
What's the difference with `CLANGNUM`/`GCCNUM` defined above? Also the two variables are equal while they have a different name. This is a bit confusing.

Reply:
a lot of this was some placeholder for a potential future where the compiler could be set during make and isn't used